src/Service/SF1500/SF1500.php

```diff
@@ -0,0 +1,859 @@
+<?php
+
+/**
+ * This file is part of itk-dev/serviceplatformen.
+ *
+ * (c) 2020 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace ItkDev\Serviceplatformen\Service\SF1500;
+
+use ItkDev\Serviceplatformen\Certificate\CertificateLocatorInterface;
+use ItkDev\Serviceplatformen\Service\SF1514\SF1514;
+use ItkDev\Serviceplatformen\Service\SoapClient;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+use Symfony\Contracts\Cache\CacheInterface;
+
+class SF1500
+{
+    private SoapClient $client;
+    private PropertyAccessor $propertyAccessor;
+    private SF1500XMLBuilder $xmlBuilder;
+    private SF1514 $sf1514;
+    private array $options;
+
+    public function __construct(SoapClient $client, SF1514 $sf1514, SF1500XMLBuilder $xmlBuilder, PropertyAccessor $propertyAccessor, array $options)
+    {
+        $this->client = $client;
+        $this->sf1514 = $sf1514;
+        $this->xmlBuilder = $xmlBuilder;
+        $this->propertyAccessor = $propertyAccessor;
+        $resolver = new OptionsResolver();
+        $this->configureOptions($resolver);
+        $this->options = $resolver->resolve($options);
+    }
+
+    /**
+     * Fetches person name from SF1500.
+     */
+    public function getPersonName(string $brugerId): string
+    {
+        $token = $this->fetchSAMLToken();
+
+        if (null === $token) {
+            return '';
+        }
```

Comment:
Not being able to get a token should result in an exception being thrown.

Reply:
Fixed to throw exception.